### PR TITLE
Move to using new Focus traversal classes/apis and remove references to deprecated APIs

### PR DESCRIPTION
**Pull request by perclasson**
This is the same changes that was done in @rami-a gallery-master branch in flutter/samples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,27 @@ wild.
 
 ![Flutter Gallery](https://user-images.githubusercontent.com/6655696/73928238-0d7fcc80-48d3-11ea-8a7e-ea7dc5d6e713.png)
 
+## Running Flutter Gallery on Flutter's master channel
+
+The Flutter Gallery target Flutter's master channel. As such it can take advantage
```
Comment (guidezpl) [inline README.md]: typo: targets

Comment (guidezpl) [reply README.md]: As such,

File: README.md [hunk 2]
```diff
@@ -8,6 +8,26 @@ wild.
...
+of new SDK features that haven't landed in the stable channel.
+
+If you'd like to run the Flutter Gallery, make sure to switch the the master channel
```
Comment (guidezpl) [inline README.md]: the the

File: README.md (hunk 1 above)
Comment (perclasson) [reply README.md]: Done!

File: README.md [hunk 3]
```diff
@@ -8,6 +8,26 @@ wild.
...
+```bash
+flutter channel master
+flutter upgrade
+```
+
+When you're done, use this command to return to the safety of the stable
+channel:
+
+```bash
+flutter channel stable
```
Comment (guidezpl) [inline README.md]: maybe add a `flutter upgrade` here too?

File: README.md (hunk 1 above)
Comment (perclasson) [reply README.md]: Done again :)

File: README.md (hunk 2 above)
Comment (perclasson) [reply README.md]: changed to: `to the`

File: README.md (hunk 3 above)
Comment (perclasson) [reply README.md]: Sure!

**Pull request merged**